assignments/function-conversion.js

```diff
@@ -13,16 +13,25 @@ myFunction();
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => param;
```

Comment:
Great job! 

When you use an arrow function and have one param, you don't need the ().

Reply:
Ok